### PR TITLE
stolostron/backlog#21143 upd vols serv

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: services/volsync/volsync_replicate.adoc [hunk 1]
```diff
@@ -23,32 +21,87 @@ Before installing VolSync on your clusters, you must have the following requirem
...
+----
++
+The VolSync service pod is installed on the managed cluster. 
+
+. Add the same label the other managed cluster. 
+
+. Run the following command on each managed cluster to make sure that the VolSync pod is running:
++
+----
+oc get pods
```
Comment (tesshuflower) [inline services/volsync/volsync_replicate.adoc]: Maybe we could change this to a command to confirm the volsync operator is installed?  We could get them to run this:

```
oc get csv -n openshift-operators
```

And they should see an entry for VolSync.

File: services/volsync/volsync_replicate.adoc [hunk 2]
```diff
@@ -23,32 +21,87 @@ Before installing VolSync on your clusters, you must have the following requirem
...
-. Apply the policy while you are logged in to your hub cluster by entering the following command:
+Replace `managed-cluster-1-namespace` with the namespace of one of your managed clusters. This namespace is the same as the name of the managed cluster.
++
+*Note:* The name must be `volsync`. 
+
+. Apply the file to your configuration by entering a command similar to the following example:
 +
 ----
-oc -n <namespace> apply -f ./policy-persistent-data-management.yaml
+oc apply volsync-mcao.yaml
```
Comment (tesshuflower) [inline services/volsync/volsync_replicate.adoc]: We'll need the -f flag since applying from a yaml file.

```
oc apply -f volsync-mcao.yaml
```

**Comment by cadawson** [comment]
Thanks @tesshuflower . Updated the content with the suggested changes. Let me know if we are ready to merge this one. Thanks!

**Review by tesshuflower — APPROVED** [review]
/lgtm

Thanks Chris!

**Comment by cadawson** [comment]
Merging. Can move, if necessary.

**Pull request merged**